modules/define.py

```diff
@@ -1,24 +1,42 @@
 #--require-config wordnik-api-key
 
 import Utils
+import time
 
-URL_WORDNIK = "http://api.wordnik.com:80/v4/word.json/%s/definitions"
+URL_WORDNIK = "https://api.wordnik.com/v4/word.json/%s/definitions"
+URL_WORDNIK_RANDOM = "https://api.wordnik.com/v4/words.json/randomWord"
+
+RANDOM_DELAY_SECONDS = 3
 
 class Module(object):
     def __init__(self, bot, events, exports):
         self.bot = bot
+        self.last_called = 0
```

Comment:
just a very small problem here but it's about python code styling. things that shouldn't be referenced outside the module (e.g. `private` in languages that support it) should be prefixed with a `_` to visually denote this as python doesn't support *actual* private.